transformation_test/transformation_test.go

```diff
@@ -569,7 +569,7 @@ func sanitizeWriteLogEntriesRequest(t *testing.T, r *logpb.WriteLogEntriesReques
 func sanitizeStacktrace(t *testing.T, input string) string {
 	// We need to remove non-deterministic information from stacktraces so the goldens don't keep changing.
 	// Remove $GOPATH
-	result := regexp.MustCompile(`(?m)^\t(.*?)pkg/mod/`).ReplaceAllString(input, "\t")
+	result := regexp.MustCompile(`(?m)^\t(.*?)pkg/mod/`).ReplaceAllString(input, "  ")
```

Comment:
I'm curious, why did it has to change from tabs (`\t`) to two spaces ?

Reply:
Using `\t` (tab) where we did is illegal according to the YAML spec. Tabs can't be used as indentation, and putting the `\t` character where we did ended up mixing space and tab indentation. The old version of the YAML library parsed this incorrectly, allowing it to be possible when it should be illegal. This is fixed in this newer version of the library. I decided on two spaces to maintain the original intent of the replacement while still making it legal YAML.